### DISCUSSION TITLE
RaZberry 7 and RaZberry 7 Pro by z-wave.me Z-Wave 700 Series shields looks like alternatives to Aeotec Z-Pi 7, Aeotec Z-Stick 7, Zooz USB ZST10 700 Stick, and Silabs UZB-7

### DISCUSSION THREAD

**Discussion by Hedda** [zwave-js/zwave-js#4112] [General]
FYI, z-wave.me have now released two Z-Wave 700 Series (Z-Wave Gen7 / Z-Wave Plus V2) shield products for Raspberry Pi GPIO header.

One larger PCB variant with an onboard circuit-board antenna and one smaller PCB "PRO" variant which has an external antenna.

https://z-wave.me/products/razberry/

https://z-wave.me/wa-data/public/site/themes/epsilon/img/cards_1.jpg

![image](https://user-images.githubusercontent.com/6320001/150318667-f89ac23b-c8a8-42fe-ba0e-b175e49a2a16.png)

RaZberry 7 with a built-in PCB antenna and based on based on Silicon Labs ZGM130S (Silabs SiP radio module with EFR32ZG14 MCU)
ZME_RAZBERRY7 (global/international version for all frequency range via different firmware releases)

RaZberry 7 Pro with uFL anenna connector for external SMT antenna also based on Silicon Labs ZGM130S (SiP module with EFR32ZG14)
- ZMEERAZBERRY7_ANT (model with external antenna for 908–917 or 919–921 MHz)
- ZMEURAZBERRY7_ANT (model with external antenna for 865–869 MHz)

https://rus.z-wave.me/drive/?file=24e4716409e82d0e27379e3e5208c8667b99

https://manuals.plus/m/4dface0320168cc13cc99e53913d0c4862f3d61931de716c60ff1d284d7aa79f_optim.pdf

https://manuals.plus/z-wave-me/razberry-7-pro-z-wave-shield-with-external-antenna-manual

https://z-wave.me/wa-data/public/site/themes/epsilon/img/1698.970.jpg

![image](https://user-images.githubusercontent.com/6320001/150319232-b49b498b-dd0e-47be-8a65-3d35a662241c.png)

https://z-wave.me/wa-data/public/site/themes/epsilon/img/1695.970.jpg

![image](https://user-images.githubusercontent.com/6320001/150319254-6a1f2daf-5cde-43fd-8837-b9a3b556bbeb.png)

Not sure about price but these could be alternatives to Aeotec Z-Stick 7 USB stick (ZWA010), Aeotec Z-Pi 7 Raspberry Pi HAT/Shield (ZWA025), 
Zooz USB 700 Series Z-Wave Plus S2 Stick ZST10 700, and Silicon Labs UZB-7 USB Stick (Silabs SLUSB001A / SLUSB7000A):

https://github.com/zwave-js/node-zwave-js/discussions/2549

https://github.com/zwave-js/node-zwave-js/discussions/2544

https://github.com/zwave-js/node-zwave-js/discussions/3409

https://github.com/zwave-js/node-zwave-js/discussions/4055

https://github.com/zwave-js/node-zwave-js/discussions/2927

PS: Note that there is currently a serious problem with Z-Wave 700 Series controllers in node-zwave-js / node-zwavejs2mqtt, see this issue:

https://github.com/zwave-js/node-zwave-js/issues/3906

**Comment by Hedda** [comment]
Based on picture of the RaZberry 7 with built-in PCB antenna it looks like it will not fit in “Home Assistant Yellow” / “Home Assistant Amber”:

https://www.crowdsupply.com/nabu-casa/home-assistant-yellow/updates/z-wave-and-rename

![image](https://user-images.githubusercontent.com/6320001/143401380-ab664d22-69de-4ad9-8296-2ef391899614.png)

### _Z-Wave and Rename_

_Today we are announcing the rename to Home Assistant Yellow. It will stay the same product we announced as Home Assistant Amber, it’s just a new name._

_A couple of weeks back Lewis from Everything Smart Home asked me if Aeotec’s Z-Pi 7 would work with Home Assistant Yellow. We have a subset of the GPIO pins available on the J11 header, which offers plenty of 3.3 V power and a UART directly connected to the Compute Module 4. Electronically, this is all what is required to make the Z-Pi 7 work._

_We received two Z-Pi 7 from our friends at Aeotec and can confirm: Z-Pi 7 works with Home Assistant Yellow and the Z-Wave JS Add-on and Integration. It also fits into the case which provides an easy and clean way to extend Home Assistant Yellow with Z-Wave support._

_We haven’t measured or compared RF performance yet. The orientation of the antenna might not be quite ideal, but keep in mind that Z-Wave forms a mesh-network. Z-Wave also uses a different frequency spectrum than the onboard Zigbee radio, hence interference should be minimal._

![image](https://user-images.githubusercontent.com/6320001/143401397-f1859fd7-4e23-4c5e-b937-617f141e0c45.png)

**Reply by PoltoS** [reply]
It will. They moved the port from the border to let Razberry 7 fit in

**Comment by Hedda** [comment]
FYI, here are direct links for upstream Silicon Labs Z-Wave 7.17.2 reference firmware for ZGM130S in EU and US regions. Not custom as Zwave.me default but should work on Z-Wave.Me RaZberry 7 / Z-Wave.Me RaZberry 7 Pro (zme_razberry7 by Smart Systems Ltd. Z-Wave.Me:

https://github.com/SiliconLabs/gecko_sdk/blob/gsdk_4.0/protocol/z-wave/Apps/bin/gbl/ZW_SerialAPI_Controller_7_17_2_406_ZGM130S_REGION_EU.gbl?raw=true

https://github.com/SiliconLabs/gecko_sdk/blob/gsdk_4.0/protocol/z-wave/Apps/bin/gbl/ZW_SerialAPI_Controller_7_17_2_406_ZGM130S_REGION_US.gbl?raw=true

As mentioned before, Silicon Labs are now releasing on GitHub so other regions and different chips can also be found by selecting "Go to file"  on https://github.com/SiliconLabs/gecko_sdk/tree/gsdk_4.0/protocol/z-wave/Apps/bin/gbl and then entering "`ZW_SerialAPI_Controller_7_17_2_406_`" (for web download must then go to the GiHub web page first and then click "Download").

**Reply by AlCalzone** [reply]
ZWave.me have their **own firmwares**, which are based on the official SDK but add additional features. Use this information above with caution.

**Reply by AndreBremer** [reply]
ZWave.me uses a custom bootloader, so the minicom approach doesn't work with Razberry7. The only working method to update firmware involves using Z-Way, but it only appears to work with ZWave.me's own firmware. Attempts to use standard firmware with this method fail. The latest supported official firmware version is 7.13.

**Reply by Botched1** [reply]
7.13? Yikes. That sounds like a great reason for me to not buy one of those then. Thanks!

**Reply by PoltoS** [reply]
@Botched1 7.13 is the app version, not the SDK version. Please don't be confused. 7.13 app version uses 7.17.02 SDK!

**Reply by Botched1** [reply]
Good point, I wasn't thinking about that - I just took the comment at face value.

Thanks!
